indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorSpec.java

```diff
@@ -50,6 +50,7 @@
   protected final SeekableStreamIndexTaskClientFactory indexTaskClientFactory;
   protected final ObjectMapper mapper;
   protected final RowIngestionMetersFactory rowIngestionMetersFactory;
+  private final SeekableStreamSupervisorIngestionSpec ingestionSchema;
   private final DataSchema dataSchema;
```

Comment:
This class now has duplicate information between `ingestionSchema` and `dataSchema`, `ioConfig`, and `tuningConfig`. I suggest to remove `dataSchema`, `ioConfig`, and `tuningConfig` and keep only `ingestionSchema`.

Reply:
These will still be required in subclasses `KafkaSupervisorSpec` and `KinesisSupervisorSpec` for backwards compatibility, so there will be some duplication.

Reply:
It doesn't have to be duplicate. You can always create ingestionSchema and get things from it.

Reply:
ok right, removed the extra configs from this class.